### ISSUE TITLE
Crashed on column name with "."

### ISSUE THREAD

**Issue by happykhan** [phac-nml/irida#313]
![image](https://user-images.githubusercontent.com/1452696/54299803-88e9d880-45b3-11e9-89c5-a7c7ca0e64a9.png)

And it freaked out

![image](https://user-images.githubusercontent.com/1452696/54299875-af0f7880-45b3-11e9-9e81-1d5c51a15eeb.png)

It's fine if I remove the period at the end,

**Comment by apetkau** [comment]
Thanks for reporting @happykhan. @joshsadam do you have an idea of what's going on?

**Comment by joshsadam** [comment]
I can look at fixing this.  It is an issue with the way DataTables looks up the value for a cell.  Unless specified, it will use the text from the header as the key to the JSON object container the data for that row.  If there is a `.` in it, it considers it a nested key for a complex JSON object. 

## Example:
```json
{
  location: {
    country: "Canada",
    city: "Winnipeg"
  }
}
```
If the header was `location.city` the cell would display `Winnipeg`.

In this case, what is essentially be passed would be the equivalent to `location.` which is not a valid lookup.

**Comment by tom114** [comment]
Fixed in #314
